pallets/xst/src/lib.rs

```diff
@@ -181,12 +176,79 @@ pub mod pallet {
 
         #[pallet::weight(<T as Config>::WeightInfo::enable_synthetic_asset())]
         pub fn enable_synthetic_asset(
```

Comment:
Looks like we should split this function to `enable_synthetic_asset` and `register_synthetic_asset`. Because it look like it's impossible to re-enable disabled synthetic asset